tests/tensor_search/integ_tests/test_embed.py

```diff
@@ -102,22 +108,38 @@ def setUpClass(cls) -> None:
             tensor_fields=["text_field_1", "text_field_2", "image_field_1"]
         )
 
+        structured_image_index_with_e5_small = cls.structured_marqo_index_request(
+            model=Model(name='hf/e5-small'),
```

Comment:
It's probably better to use the test prefix model for these tests, since if we decide to change the `hf/e5-small` default prefixes, these tests will break.